charts/logzio-telemetry/templates/podmonitor.yaml

```diff
@@ -1,18 +0,0 @@
-{{- if and .Values.traces.enabled .Values.podMonitor.enabled .Values.podMonitor.metricsEndpoints .Values.agentCollector.enabled }}
```

Comment:
We delete this template but the values file still contains the deprecated values